app/src/test/java/org/mozilla/fenix/home/DefaultSessionControlControllerTest.kt

```diff
@@ -357,6 +363,120 @@ class DefaultSessionControlControllerTest {
         verify { activity.openToBrowser(BrowserDirection.FromHome) }
     }
 
+    @Test
+    fun handleSelectGoogleDefaultTopSiteUS() {
+        val topSiteUrl = SupportUtils.GOOGLE_URL
+
+        every { store.state.search.region } returns RegionState("US", "US")
```

Comment:
nit: My experience with refactorings so far was that mocking to many internals is rather painful when changing code. I'd either dispatch an action to change the region here, or alternatively create a store with the right state in every test method. But that's of course not blocking here.